### PR TITLE
Update outdated class names in the docs

### PR DESCRIPTION
**Pull request by jdreesen**

### CODE REVIEW

File: doc/Development_Documentation/05_Objects/01_Object_Classes/01_Data_Types/10_Calculated_Value_Type.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ You can also provide a Symfony service as calculator class via `@` prefix (e.g.
 
 Getter methods on the object class are generated as usual. The code to retrieve the values would then be: 
 ```php
-$object = Object_Abstract::getByPath("/demoobject");
+$object = DataObject::getByPath("/demoobject");
```
Comment (fashxp) [inline doc/Development_Documentation/05_Objects/01_Object_Classes/01_Data_Types/10_Calculated_Value_Type.md:67]: Should be use `AbstractObject` here, as `DataObject` is just an alias, wdyt?

File: doc/Development_Documentation/05_Objects/01_Object_Classes/05_Class_Settings/70_Variants.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ are shown in a grid. Via buttons object variants can be created, opened and dele
 
 
 To create object variants via code, just create a normal object, set as parent the generic t-shirt and set the object 
-type to `Object_Abstract::OBJECT_TYPE_VARIANT`.
+type to `DataObject::OBJECT_TYPE_VARIANT`.
```
Comment (dvesh3) [inline doc/Development_Documentation/05_Objects/01_Object_Classes/05_Class_Settings/70_Variants.md:35]:
```suggestion
type to `AbstractObject::OBJECT_TYPE_VARIANT`.
```

File: doc/Development_Documentation/18_Tools_and_Features/05_Notes_and_Events.md [hunk 3]
```diff
@@ -30,7 +30,7 @@ $note->setUser(0);
  
 // you can add as much additional data to notes & events as you want
 $note->addData("myText", "text", "Some Text");
-$note->addData("myObject", "object", Object_Abstract::getById(7));
+$note->addData("myObject", "object", DataObject::getById(7));
```
Comment (dvesh3) [inline doc/Development_Documentation/18_Tools_and_Features/05_Notes_and_Events.md:33]:
```suggestion
$note->addData("myObject", "object", Model\DataObject::getById(7));
```

File: doc/Development_Documentation/18_Tools_and_Features/05_Notes_and_Events.md [hunk 4]
```diff
@@ -30,7 +30,7 @@ $note->setUser(0);
...
 $note->addData("myDocument", "document", Document::getById(18));
```
Comment (dvesh3) [inline doc/Development_Documentation/18_Tools_and_Features/05_Notes_and_Events.md:34]:
```suggestion
$note->addData("myDocument", "document", Model\Document::getById(18));
```

File: doc/Development_Documentation/18_Tools_and_Features/05_Notes_and_Events.md [hunk 5]
```diff
@@ -30,7 +30,7 @@ $note->setUser(0);
...
 $note->addData("myAsset", "asset", Asset::getById(20));
```
Comment (dvesh3) [inline doc/Development_Documentation/18_Tools_and_Features/05_Notes_and_Events.md:35]:
```suggestion
$note->addData("myAsset", "asset", Model\Asset::getById(20));
```

File: doc/Development_Documentation/05_Objects/01_Object_Classes/01_Data_Types/10_Calculated_Value_Type.md (hunk 1 above)
Comment (jdreesen) [reply doc/Development_Documentation/05_Objects/01_Object_Classes/01_Data_Types/10_Calculated_Value_Type.md:67]: Sure, if that's preferred.

**Comment by brusch** [comment]
Thanks a lot! 👍

**Pull request merged**